### PR TITLE
[WithTest]Todo items

### PR DESCRIPTION
**Pull request by 5eeya**
[WithTest]Todo items

### CODE REVIEW

**Comment by zongyi001** [comment]
Good! commit message is very clear

File: TodoItems.Core/TodoItem.cs [hunk 1]
```diff
@@ -1,9 +1,54 @@
-﻿namespace TodoItems.Core;
+﻿using static System.Runtime.InteropServices.JavaScript.JSType;
 
-public class TodoItem
+namespace TodoItems.Core;
+
+public class TodoItem 
 {
+    private readonly ItodosRepository _todosRepository;
+    public List<DateTime>? TimeStamps;
```
Comment (kgaohahaha) [inline TodoItems.Core/TodoItem.cs]: One Item can modify 3 times in a day

File: TodoItems.Test/TodoItemTest.cs [hunk 2]
```diff
@@ -7,7 +9,182 @@ public class TodoItemTest
...
+            Description = "new task",
+            IsDone = true,
+            IsFavorite = true,
+            DueDate = new DateTime(2024, 10, 31),
+            CreatedDate = new DateTime(2024, 10, 30)
+        };
+        var TodoItems = new List<TodoItem> { };
+        for (int i = 0; i < 6; i++)
+        {
+            TodoItems.Add(new TodoItem(mockRepository.Object));
```
Comment (kgaohahaha) [inline TodoItems.Test/TodoItemTest.cs]: new TodoItemDto()

**Comment by xYYux** [comment]
commit message highlight Class name,  good job

**Pull request closed without merging**